src/OVAL/probes/probe/main.c

```diff
@@ -228,7 +228,11 @@ int main(int argc, char *argv[])
 	 * Initialize SEAP stuff
 	 */
 	probe.SEAP_ctx = SEAP_CTX_new();
-	probe.sd       = SEAP_openfd2(probe.SEAP_ctx, STDIN_FILENO, STDOUT_FILENO, 0);
+#ifdef _WIN32
+	probe.sd = SEAP_openfd2(probe.SEAP_ctx, _fileno(stdin), _fileno(stdout), 0);
```

Comment:
What about defining the `STDIN_FILENO` etc. for W32 close to the include directive? [stacko says](https://stackoverflow.com/questions/15102992/what-is-the-difference-between-stdin-and-stdin-fileno) that it is defined in `unistd.h` on unixes.